app/services/system_metrics.rb

```diff
@@ -2,8 +2,18 @@
 
 require 'sidekiq/api'
 
-class GetSystemMetrics
+class SystemMetrics
+  LATENCY_THRESHOLDS = {
+    'default' => 1,
+    'short_update' => 2.hours,
+    'medium_update' => 12.hours,
+    'long_update' => 1.day,
+    'daily_update' => 1.day,
+    'constant_update' => 15.minutes
+}.freeze
```

Comment:
This is probably a linting violation, it should be indented the same as the constant name.

I usually also place a constant like this, that is used by one specific method, right above the method that uses it. That makes it a little easier to browse the code and see what the constant is for.